internal/testutil/prototest/prototest.proto

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Fraunhofer AISEC
```

Comment:
Maybe get rid of that file and minimize the generated go file. But on the other side it is good if sth. changes with proto messages and their generations

Reply:
But in general, I like the approach very much :)

Reply:
If I delete that file we have the same problem as before. In internal/util/struct_test.go we need a proto struct for the test. 

I see no problem in having this file and would let it as it is.

Reply:
If you basically just need "any" protobuf message for testing, you can actually use EmptyPb for example.

Reply:
You nee a protobuf message with field names, I don't think that an EmtyPb is enough!?!